services/ux-backend/handlers/common.go

```diff
@@ -1,5 +1,21 @@
 package handlers
 
+import "os"
+
 const (
 	ContentTypeTextPlain = "text/plain"
 )
+
+var namespace string
+
+// returns namespace found in pod
+func GetPodNamespace() string {
+	if namespace != "" {
+		return namespace
+	}
+	if ns := os.Getenv("OPERATOR_NAMESPACE"); ns != "" {
+		namespace = ns
+		return namespace
+	}
+	panic("Value for env var 'POD_NAMESPACE' is empty")
+}
```

Comment:
This function is already defined in utils let's use that

Reply:
removed the dublicate code, used fro utils